docs/public/specifications/tests/Basic-TP/basic01.md

```diff
@@ -64,9 +64,9 @@ B01_INCONSISTENT_ALIAS     |ERROR  | domain                                | The
 B01_INCONSISTENT_DELEGATION|ERROR  |domain_child, domain_parent, ns_ip_list| The name servers for parent zone "{domain_parent}" give inconsistent delegation of "{domain_child}". Returned from name serververs "{ns_ip_list}".
 B01_NO_CHILD               |ERROR  | domain_child, domain_super            | "{domain_child}" does not exist as a DNS zone. Try to test "{domain_super}" instead.
 B01_PARENT_FOUND           |INFO   | domain, ns_ip_list                    | The parent zone is "{domain}" as returned from name servers "{ns_ip_list}".
+B01_PARENT_NOT_FOUND       |WARNING|                                       | The parent zone cannot be found.
 B01_PARENT_UNDETERMINED    |WARNING| ns_ip_list                            | The parent zone cannot be determined on name servers "{ns_ip_list}".
-B01_UNEXPECTED_NS_RESPONSE |WARNING|domain_child, domain_parent, ns_ip_list| Name servers for parent domain "{domain_parent}" give an incorrect response on SOA query for "{domain_child}". Returned from name servers "{ns_ip_list}".
-
+B01_SERVER_ZONE_ERROR      |DEBUG  | query_name, rrtype, ns_ip             | Unexpected response on "dig {query_name} {rrtype} @{ns_ip}"
```

Comment:
I don’t think we should refer to `dig` in our messages. And “unexpected” doesn’t clarify the problem with the response.

Maybe: “Name server {ns_ip} gave an erroneous response to a query of type {query_type} for {query_name}”? Or some variant? This wording isn’t perfect, though, because this message is also emitted when the server fails to respond or responds with NOERROR but with the AA bit unset.